parser.go

```diff
@@ -252,7 +255,55 @@ func (p *parser) readValue(in []byte,
 		if strings.Contains(line, `\#`) {
 			line = strings.Replace(line, `\#`, "#", -1)
 		}
+	} else if lastChar == '\n' && allowPythonMultilines {
```

Comment:
I think it would be more efficient if we check `allowPythonMultilines` before `lastChar == '\n'`, e.g. `else if allowPythonMultilines && lastChar == '\n'`.